dotnet/src/Functions/Functions.OpenAPI/Builders/Serialization/SimpleStyleParameterSerializer.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Text.Json.Nodes;
+using Microsoft.SemanticKernel.Plugins.OpenApi.Model;
+
+namespace Microsoft.SemanticKernel.Plugins.OpenApi.Builders.Serialization;
```

Comment:
nit: the class is internal, but maybe still worth to keep namespaces short.

Reply:
Made it a bit shorter.